util/format.js

```diff
@@ -16,4 +16,16 @@ module.exports = (deck) => {
     }
     cardIndex++;
   }
+  log(horizontalLine());
 };
+
+module.exports.printCard = (card) => {
+  if (!card) {
+    return log(`No card found.`);
+  }
+  log(horizontalLine());
+  log(`${bold('Influence:')} ${card.name}`);
```

Comment:
Is there any way you can colour the output?

Reply:
Not without modifying the game object data, will open another issue for that.

Reply:
See #45